src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,89 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Libre Baskerville', serif;
+  font-size: 25px;
+}
+
+.container {
+  height: 100vh;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stopwatch {
+  position: relative;
+  @include watch(300px);
+
+  border: 10px solid rgb(161, 93, 161);
+  background-color: rgb(245, 180, 210);
```

Comment:
would be nice to add variables for colors like that, it really looks difficult, and you want it clear. 
Not critical in this task though